README.md

```diff
@@ -118,20 +118,3 @@ try {
 * If you specify a stored template and also provide inline content via `html` or `text`, you will receive an error.
 * By default, open and click tracking are enabled for a transmission.
 * By default, a transmission will use the published version of a stored template.
-
-## Development
-
-### Setup
-Run `composer install` inside the directory to install dependecies and development tools.
-
-### Testing
-Once all the dependencies are installed, you can execute the unit tests using:
-```
-composer test
-```
-
-### Contributing
```

Comment:
Probably worth it to keep a Contributing header and just say `See CONTRIBUTING` where CONTRIBUTING links to the file.